### PR TITLE
create quarterly report script

### PR DESCRIPTION
**Pull request by jonathanadler**
We can improve this of course

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_610/

File: scripts/quarterlyReport.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const simpleGit = require('simple-git/promise');
+const chalk = require('chalk');
+
+const sGit = simpleGit();
+
+const [, , startDate, endDate] = process.argv;
```
Comment (sivanharel) [inline scripts/quarterlyReport.js:7]: The two ',' are a bit confusing. Maybe it is better to write the names of these vars even we are not using them..

**Review by sivanharel — APPROVED** [review]
Cool:)

File: scripts/quarterlyReport.js (hunk 1 above)
Comment (jonathanadler) [reply scripts/quarterlyReport.js:7]: Yeah, maybe it will be easier to use some command line tool for this

**Pull request merged**